src/run/handlers/cache.cts

```diff
@@ -275,7 +289,7 @@ export class NetlifyCacheHandler implements CacheHandler {
       }),
     )
 
-    purgeCache({ tags }).catch((error) => {
+    await purgeCache({ tags }).catch((error) => {
```

Comment:
this doesn't change much, because `.revalidateTag` overall was not awaited on and all of this is ~background work that won't block response